qemu/tests/cfg/power_nvram.cfg

```diff
@@ -0,0 +1,22 @@
+- power_nvram:
+    type = power_nvram
+    start_vm = yes
```

Comment:
Can set this "start_vm = no" and delete "start_vm = no" for big_size and bigger_size

Reply:
Thanks for your reviewing but we need to set it as "yes" since the last case (normal size) need it.

Reply:
I agree with @maxujun , 2/3 use `start_vm = no`, only one use ` = yes`. So we can define `start_vm = no` as the public parameter, just add `start_vm = yes` in the specific variant.